builder/plan.sh

```diff
@@ -1,7 +1,7 @@
 pkg_origin="core"
 pkg_name="builder"
 pkg_type="composite"
-pkg_version="0.1.0"
+pkg_version="0.1.1"
```

Comment:
I don't think you need to bump this. We should just be using the same technique to determine version as the other components (`git rev-list master --count`)